### PR TITLE
parse port ranges

### PR DESCRIPTION
**Pull request by uvgroovy**
Added parsing port range utility.
to assist with:
https://github.com/docker/compose/issues/1102
see:
https://github.com/docker/compose/pull/1191#discussion_r27248067
for more details.

### CODE REVIEW

File: docker/utils/ports/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+from .ports import (
+    to_port_range,
+    split_port,
+    add_port_mapping,
+    build_port_bindings
```
Comment (aanand) [inline docker/utils/ports/__init__.py]: I don't think `to_port_range` and `add_port_mapping` should be imported here - they're only used within `ports.py`, and I can't see them being useful outside it

**Comment by aanand** [comment]
Looks good, apart from minor API surface area issue (comment inline).

**Comment by uvgroovy** [comment]
@aanand  - thanks for the feedback - fixed

**Comment by shin-** [comment]
LGTM, thanks for taking the time to do this!

**Pull request merged**